components/proxy/src/main/java/com/hotels/styx/ResponseInfoFormat.java

```diff
@@ -13,20 +13,20 @@
   See the License for the specific language governing permissions and
   limitations under the License.
  */
-package com.hotels.styx.proxy;
-
-import com.hotels.styx.Environment;
+package com.hotels.styx;
 
 import static com.hotels.styx.StyxConfig.NO_JVM_ROUTE_SET;
 import com.hotels.styx.api.LiveHttpRequest;
 
 /**
  * Formats response info into a string.
  */
+
+// TODO: Should be package private, but it is shared by com.hotels.styx.servers.StyxHttpServer.kt
```

Comment:
Not sure we need a TODO here. It's a trivial problem and the TODO will be there forever and will never get fixed. Better would be to fix it by moving StyxHttpServer to com.hotels.styx package or just remove comment.